django_ipam/widgets.py

```diff
@@ -0,0 +1,11 @@
+from django import forms
+from django.contrib.admin.templatetags.admin_static import static
+from django.contrib.admin.widgets import RelatedFieldWidgetWrapper
+
+
+class IpRequestWidget(RelatedFieldWidgetWrapper):
```

Comment:
we don't need this, you can load the JS from the admin class, let's get rid of all this additional unneeded code